packages/experimental/src/contextual-help/docs/ContextualHelp.stories.mdx

```diff
@@ -20,10 +20,21 @@ import { Inline } from "@orbit-ui/components";
 
 ## Guidelines
 
+### Accessibility
+
+This components is tabbable and can be focused with the keyboard.
+
 ### HelpMessage vs. ContextualHelp
 
 Use [HelpMessage](?path=/docs/field--input#input) for critical information that a user needs to know to complete a task. Don't hide essential information in ContextualHelp; it's intended to supplement the experience with minimal disruption.
 
+### Note
+
+We should consider converting this component from a tooltip to a button with a popover for 3 reasons:
+- Tooltips don't work in mobiles/tablets
```

Comment:
- Tooltips don't work well on touch devices

they work sometimes but it's non discoverable since they appear on click